float32 current_time
float32 measurement_time
int32 remote_address
geometry_msgs/Vector3 xyz
geometry_msgs/Vector3 enu
geometry_msgs/Vector3 rpy
float32 propagation_time
int32 rssi
int32 integrity
float32 accuracy


